remo/reports/api/views.py

```diff
@@ -19,3 +30,77 @@ def retrieve(self, request, pk):
         serializer = ActivitiesDetailedSerializer(report,
                                                   context={'request': request})
         return Response(serializer.data)
+
+
+class ActivitiesKPIFilter(django_filters.FilterSet):
+    """Filter for activities KPI endpoint."""
+    category = django_filters.CharFilter(name='functional_areas__name')
+    initiative = django_filters.CharFilter(name='campaign__name')
+
+    class Meta:
+        model = NGReport
+        fields = ['country']
+
+
+class ActivitiesKPIView(APIView):
+
+    def get(self, request):
+        """Returns serialized data for Activities KPI"""
+
+        queryset = NGReport.objects.filter(report_date__lte=now())
+        activities = ActivitiesKPIFilter(request.GET, queryset=queryset)
+        weeks = int(request.query_params.get('weeks', KPI_WEEKS))
+
+        # Total calculations
+        total = activities.qs.count()
+
+        # Quarter calculations
+        current_quarter = int(math.ceil(now().month/3.0))
+        current_quarter_start = datetime(now().year, current_quarter, 1)
```

Comment:
You are right. This fell between the cracks since the tests are refering to the first quarter. I am fixing this in both activities/reports.